### PR TITLE
Some modified

### PR DESCRIPTION
**Pull request by pyonk**
This package can not working, so I fixed it using "shlex".
And, I've added "server stop" command and "deploy github pages" command.

Thank you.

### CODE REVIEW

File: Hugofy.sublime-keymap [hunk 1]
```diff
@@ -4,6 +4,8 @@
     <binding key="ctrl+alt+3" command="hugogetthemes"/>
     <binding key="ctrl+alt+4" command="hugosettheme"/>
     <binding key="ctrl+alt+5" command="hugobuild"/>
-    <binding key="ctrl+alt+6" command="hugoserver"/>
-    <binding key="ctrl+alt+7" command="hugoversion"/>
-</bindings>
\ No newline at end of file
+    <binding key="ctrl+alt+6" command="hugoserverstart"/>
+    <binding key="ctrl+alt+7" command="hugoserverstart"/>
```
Comment (akmittal) [inline Hugofy.sublime-keymap:8]: It should be hugoserverstop

File: Hugofy.py [hunk 2]
```diff
@@ -31,58 +32,102 @@ def run(self,edit):
...
+		theme = settings.get("DefaultTheme")
 		try:
 			startCmd = "hugo server --theme={} --buildDrafts --watch --port={}".format(theme, server["PORT"])
-			out=subprocess.Popen(startCmd,stderr=subprocess.STDOUT,universal_newlines=True)
+			out = subprocess.Popen(shlex.split(startCmd), stderr=subprocess.STDOUT, universal_newlines=True)
 			sublime.status_message('Server Started: {}'.format(startCmd))
 		except:
 			sublime.error_message("Error starting server")
 
+class HugoserverstopCommand(sublime_plugin.TextCommand):
```
Comment (akmittal) [inline Hugofy.py:54]: This code seems for *nix environment only. We need to support Windows also.

File: Hugofy.py [hunk 3]
```diff
@@ -31,58 +32,102 @@ def run(self,edit):
...
+	def on_change(self, themename):
 		pass
 	def on_cancel(self):
 		pass
 
+	def run(self, edit):
+		setvars()
+		sublime.active_window().show_input_panel("Enter theme name", "", self.on_Pdone, self.on_change, self.on_cancel)
+
+class HugodeployCommand(sublime_plugin.TextCommand):
```
Comment (akmittal) [inline Hugofy.py:113]: Wont work on windows

**Comment by akmittal** [comment]
Thanks for the PR.
There are few issues we need to address as i mentioned in review.

**Comment by pyonk** [comment]
Thank you for your review.
I will come back again!

**Pull request closed without merging**